File: MeetingManagement.Model/DataTransferObjects/MeetingDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MeetingManagement.Model.DataTransferObjects
{
    public class MeetingDto
    {
        public MeetingType Type { get; set; }
        public string Number { get; set; } = string.Empty;
        public string Notes { get; set; } = string.Empty;

        public long Id { get; set; } = 0;

        public IEnumerable<long> Items { get; set; }
    }
}

File: MeetingManagement.Web/src/features/meeting/models/meeting.dto.ts
export class MeetingDto{
  type = MeetingType.MANCO;
  number = "";
  notes  = "";
  id = 0;
  items: Array<number> = []
}

export const enum MeetingType
{
    MANCO = 1,
    Finance = 2,
    PTL = 3
}
